torch/_higher_order_ops/utils.py

```diff
@@ -378,3 +386,37 @@ def _stack_pytree(pytrees):
         else:
             raise RuntimeError(f"Cannot stack {leaves}.")
     return pytree.tree_unflatten(stacked_out, out_spec)
+
+
+# We cannot call save_for_backward for symints. This helper function
+# can be used to save symints as a direct attributes in ctx.
+def save_tensors_and_symints_for_backward(ctx, args):
```

Comment:
@Chillee sounds like you wanted to use this too?

Reply:
Yeah I also wanted this.